src/utils.ts

```diff
@@ -54,12 +54,16 @@ export const openPopup = () => {
   return popup;
 };
 
-export const runPopup = (popup: any, authorizeUrl: string) => {
+export const runPopup = (
+  popup: any,
+  authorizeUrl: string,
+  config: PopupConfigOptions
+) => {
   popup.location.href = authorizeUrl;
   return new Promise<AuthenticationResult>((resolve, reject) => {
     const timeoutId = setTimeout(() => {
-      reject(TIMEOUT_ERROR);
-    }, 60 * 1000);
+      reject({ ...TIMEOUT_ERROR, popup });
+    }, config.timeoutInSeconds || 60 * 1000);
```

Comment:
I see you are using `60` here and in `src/Auth0Client.ts`. Please, find a way to reuse this definition, either via a constant or file.

Reply:
done